config/samples/v1alpha1_clustertemplate.yaml

```diff
@@ -44,9 +44,9 @@ spec:
             type: string
           install-plan-approval:
             type: string
-            default: Automatic
+        type: object
       clusterInstanceParameters:
-        description: clusterInstanceParameters defines the params that are allowed in the ProvisioningRequest spec.clusterInstanceInput
+        description: clusterInstanceParameters defines the params that are allowed in the ProvisioningRequest
```

Comment:
nit: to match above:
clusterInstanceParameters defines the available parameters for cluster provisioning